src/ui/templates/search/search.hbs

```diff
@@ -36,25 +36,39 @@
         </button>
         <button {{#if useForm}}type="submit"{{else}}type="button"{{/if}}
           class="js-yext-submit yxt-SearchBar-button">
+          {{#if showLoadingIndicator}}
```

Comment:
could we move the icon handlebars into it's own partial? Fun fact, any file you make in the src/ui/templates folder is automatically registered as both a partial and template in the templates bundle! @tmeyer2115 iirc in order to fulfill the "no inline JS" csp, we're only not allowed to use inline partials, but regular partials are fine right? 
If we can't use regular partials in the search bar, we'll also need to use IconComponent instead of iconPartial (this was something I didn't think about for the iconPartial PR).